target/scripts/startup/setup-stack.sh

```diff
@@ -626,6 +652,8 @@ function _setup_dkim_dmarc
   # shellcheck disable=SC2016
   sed -i -E 's|^(non_smtpd_milters =.*)|\1 \$dkim_milter|g' /etc/postfix/main.cf
 
+  mkdir -p /etc/opendkim/keys/ && touch /etc/opendkim/SigningTable
```

Comment:
this actually belongs in the first commit; but I am too lazy to rewrite history now.. :D